students/preactive/Lesson2/series.py

```diff
@@ -7,6 +7,20 @@ def fib(n):
 # print(fib(8))
 
 
+a,b = 0,1
+def fib_gen():
+    global a, b
+    while True:
+        a, b = b, a + b
+        yield a
```

Comment:
as far as your question on yield (btw awesome that you are trying out `yield`)...

you shouldn't need globals here, this works as well:
```
def fib_gen():
    a,b = 0,1
    while True:
        a, b = b, a + b
        yield a
```

Reply:
added the test and a modifed one that leads series with 0